### DISCUSSION TITLE
Are there any references on how to setup tests and fixtures with this library?

### DISCUSSION THREAD

**Discussion by VaZark** [lorinkoz/django-pgschemas#190] [General]
At the moment, I have a separate DB setup and just run pytest on a preexisting test DB. It'd be cool to be able to run tests on a completely transactional DB wth tenants being setup from zero. This would also make it easier to play nice with E2E tests with the frontend.

Are there any reference projects on how to setup the tenants?? the following gist isn't really useful as it runs on the main db instead of the test 

```python
## with :  pytest --headed --create-db --no-migrations          
@pytest.fixture(scope="session")
def django_db_setup(django_db_blocker):
    with django_db_blocker.unblock():
        management.call_command("migrate", "-s", "public")
        management.call_command("migrate", "-s", "central")
        tenant = Tenant(
            name="Test Environment",
            schema_name="testenv",
            tenant_key="TESTENV",
            frontend_url="http://localhost:3000",
            notification_email="tech@example.com",
        )
        tenant.save()
```

**Comment by lorinkoz** [comment]
You could add a new fixture that activates the tenant and returns it, e.g.

```
def tenant(db):
    tenant = Tenant.objects.first()
    with tenant:
        yield tenant
```
